src/test_rng.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cpp11.hpp>
+#include <vector>
+#include <string>
+#include <sstream>
+
+#include <dust/random/generator.hpp>
+
+template <typename T>
+std::string to_string(const T& t) {
+  std::ostringstream ss;
+  ss << t;
```

Comment:
Is this guaranteed to write out the uints to full precision (as a decimal)?

Reply:
yes, checked this and compared against the hex dump (which is just harder to look at really)